core/lib/types/src/commitment.rs

```diff
@@ -742,7 +753,8 @@ mod tests {
         expected_outputs: ExpectedOutput,
     }
 
-    #[test]
+    // TODO(X): restore this test
```

Comment:
can we restore it now? (if not - can you explain more what is broken)

Reply:
the commitment calculation is broken, I would the new correct commitment to restore the test

Reply:
probably it is worth it to just wait until the prover integration is done, then we know for sure that this commitment calculation is right